util/statvar_dcid_generator.py

```diff
@@ -0,0 +1,193 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A utility to generate dcid for statistical variables."""
+
+import copy
+import re
+
+#Constants
+_QUANTITY_REGEX = r"\[(?P<value>-|-?\d+(\.\d+)?) (?P<quantity>[A-Za-z]+)\]"
```

Comment:
should we allow '_' in addition to [A-Za-z] for quantity?

Reply:
You are right. 
Found an example: [MG_PHENYTOIN_NA/10ML500](https://datacommons.org/browser/MG_PHENYTOIN_NA/10ML500)

Thanks for pointing that out.
Will add support for '_', '/' and digits.